content/docs/Konveyor/Kai/SolutionServer/proc_tackle-enable-dev-lightspeed.md

```
@@ -0,0 +1,55 @@
+
+## Procedure
+
+1.  Log in to the your Kubernetes cluster and switch to the `konveyor-ai` project.
+
+2.  Edit the Tackle CR settings in the `tackle_hub.yml` file with the following command:
+
+        kubectl edit tackle
+
+3.  Enter applicable values for `kai_llm_provider` and `kai_llm_model` variables.
+
+  ```yaml
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)